src/frontend/efiling-frontend/src/components/page/package-confirmation/PackageConfirmation.js

```diff
@@ -197,7 +199,7 @@ export default function PackageConfirmation({
         <br />
         <br />
         {files.map((file) => (
-          <div key={file.documentProperties.name}>
+          <div key={hash(file)}>
```

Comment:
Key must be unique.  Use a hash of the object instead of a not-necessarily unique or existing name.